extensions/istio/tests/src/test/java/io/fabric8/istio/test/v1beta1/VirtualServiceTest.java

```diff
@@ -53,7 +53,7 @@
 import io.fabric8.kubernetes.client.utils.Serialization;
 import okhttp3.mockwebserver.RecordedRequest;
 
-@EnableIstioMockClient
+@EnableIstioMockClient(crud = true)
```

Comment:
Once you've enabled crud mode, you don't need to set mock server expectations. Test should pass even without `server.expect()` statements

Reply:
I've removed the `server.expect` from the `testGet` method. For the rests, I think it's necessary because we're asserting the json file (using `server.takeRequest()`).